.github/workflows/release.yml

```diff
@@ -0,0 +1,25 @@
+name: Create Release
+on:
+  push:
+    branches:
```

Comment:
We should probably filter out non-logic files

Reply:
Added:
```yaml
    paths-ignore:
      - "**/*.md"
      - "**/*.txt"
      - "**/*_test.go"
      - "LICENSE"
 ```